starter-code/scripts/articleView.js

```diff
@@ -36,15 +41,39 @@ articleView.handleCategoryFilter = function() {
   /* TODO: Just like we do for #author-filter above, we should also handle
   change events on the #category-filter element. Be sure to reset the
   #author-filter while you're at it! */
+
+  $('#category-filter').on('change', function() {
+    if ($(this).val()) {
+      /* TODO: If the select box changes to an option that has a value, we should:
+      1. Hide all of the articles
+      2. Fade in only the articles that match based on on the author
+      that was aselected. Hint: use an attribute selector to find
+      those articles that match the value, and then fade them in.
+      */
+      $('article').hide();
+      var $category = $(this).val();
+      console.log($category);
+      $('article[data-category="'+ $category +'"]').fadeIn();
+    } else {
+      /* Otherwise, we should:
+      1. Show all the articles except the template */
+      $('article').show();
+      $('')
+    }
+    $('#category-author').val('');
```

Comment:
So this isn't exactly what you what it to do. Take a look at the instructions again and check out what you are targeting